.changeset/cuddly-crabs-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+"@definitelytyped/header-parser": minor
+"@definitelytyped/dtslint": minor
```

Comment:
I failed to notice that these said `minor`. They should have been `patch`. We are `0.0.x` without stability and now we just released partial `0.1.0`s :(((((((((((((((

Reply:
Oh geez. sorry!